pages/index.tsx

```diff
@@ -58,16 +58,21 @@ const Index: React.FC<IndexPageProps> = (props: IndexPageProps) => {
               Ministry of Health NZ
             </ExternalLink>
           </p>
-          <p className="footnote">
-            Data updated{" "}
-            {DateTime.fromISO(props.dataUpdatedAtTimeUtc).toRelative()}
-          </p>
-          {props.lastCheckedAtTimeUtc ? (
-            <p className="footnote">
-              Last checked{" "}
-              {DateTime.fromISO(props.lastCheckedAtTimeUtc).toRelative()}
+          <div className="flex flex-column s:flex-row justify-content-center align-items-center footnote">
+            <p>
+              Data updated{" "}
+              {DateTime.fromISO(props.dataUpdatedAtTimeUtc).toRelative()}
             </p>
-          ) : null}
+            {props.lastCheckedAtTimeUtc ? (
+              <>
+                <span className="hidden s:visible margin-x-xs">-</span>
+                <p>
+                  Last checked{" "}
+                  {DateTime.fromISO(props.lastCheckedAtTimeUtc).toRelative()}
```

Comment:
@jishaal - forgot to ask, does this mean we can delete https://github.com/theramis/aucvid/blob/main/app/utilities/hoursBeforeNow.ts?

Reply:
> forgot to ask, does this mean we can delete

I think so. Imo its note a huge issue rather get rid of the custom helper if we don't need it. We can use `toRelative({style: "short" })` if we want a shorter string (use `min` instead of `minutes`)

Reply:
Will do in another PR (alongside renaming the is / has mounted hook)